bemani/backend/iidx/cannonballers.py

```diff
@@ -1,15 +1,2018 @@
+            },
+            {
+                'name': 'SCRATCH',
+                'id': 12,
+                'songs': [
+                    11025,
+                    16053,
+                    16031,
+                    22067,
+                ],
+            },
+            {
+                'name': 'L.E.D.-G',
+                'id': 13,
+                'songs': [
+                    15007,
+                    24000,
+                    22011,
+                    17009,
+                ],
+            },
+            {
+                'name': 'QQQ',
+                'id': 14,
+                'songs': [
+                    18062,
+                    18019,
+                    12011,
+                    16045,
+                ],
+            },
+            {
+                'name': 'BMK 2017',
+                'id': 15,
+                'songs': [
+                    24084,
+                    24017,
+                    24022,
+                    24043,
+                ],
+            },
+        ]
+
+        # Secret course definitions
+        secret_courses: List[Dict[str, Any]] = [
+            {
+                'name': 'L.E.D.-K',
+                'id': 1,
+                'songs': [
+                    13034,
+                    21068,
+                    17060,
+                    24089,
+                ],
+            },
+            {
+                'name': 'SOTA K',
+                'id': 2,
+                'songs': [
+                    16010,
+                    14038,
+                    20016,
+                    24090,
+                ],
+            },
+            {
+                'name': 'POP',
+                'id': 3,
+                'songs': [
+                    22042,
+                    14056,
+                    15003,
+                    24091,
+                ],
+            },
+            {
+                'name': 'REMO-CON',
+                'id': 4,
+                'songs': [
+                    15030,
+                    12031,
+                    22078,
+                    24092,
+                ],
+            },
+            {
+                'name': 'NUMBER',
+                'id': 5,
+                'songs': [
+                    1003,
+                    17051,
+                    17041,
+                    24093,
+                ],
+            },
+            {
+                'name': 'FANTASY',
+                'id': 6,
+                'songs': [
+                    20102,
+                    24013,
+                    23092,
+                    24094,
+                ],
+            },
+            {
+                'name': 'DRUM\'N\'BASS',
+                'id': 7,
+                'songs': [
+                    6013,
+                    22016,
+                    20073,
+                    24095,
+                ],
+            },
+        ]
+
+        # For some reason, omnimix crashes on course mode, so don't enable it
```

Comment:
Is this *still* true for CB+ omni? LOL

Reply:
Good question. Lots of comments to clean up overall probably